добавлено описание атрибутов

Gotus opened the pull request.

Commit:
добавлено описание атрибутов

Nikas10 commented on rules.txt:
```diff
@@ -38,19 +38,23 @@
-    k.Инициатива (вычисляемый)
-    l.Здоровье персонажа
-    m.Рост персонажа
+    a.Урон персонажа - определяет число очков здоровья, отнимаемых у цели при нанесении удара.
```
от чего зависит урон если персонаж голый и без пассивок?

Nikas10 commented on rules.txt:
```diff
@@ -38,19 +38,23 @@
+    a.Урон персонажа - определяет число очков здоровья, отнимаемых у цели при нанесении удара.
+    b.Заряды магии - используются для сотворения заклинаний. Имеют несколько уровней. Некоторые заклинания требуют для произнесения заряды различных уровней. 
+Могут быть восстановлены за счёт отдыха в городе или эффектов.
+    c.Защита персонажа - определяет величину, на которую уменьшается наносимый персонажу урон.
```
то же что и с уроном

Nikas10 commented on rules.txt:
```diff
@@ -38,19 +38,23 @@
+может блокировать атаки определенного списка других типов оружия. Щит позволяет блокировать урон от любого вида оружия. Измеряется в процентах. 
+    h.Шанс уклонения - определяет вероятность, с которой персонаж уклонится от удара. Для уклонения от некоторых ударов могут потребоваться особые условия. 
+Щит не позволяет уклоняться от атак. Измеряется в процентах. 
+    i.Прочность тела - определяет, насколько трудно нарушить целостность тела персонажа. Выше значение - выше прочность.
```
не совсем ясно зачем этот атрибут нужен и как он будет влиять на расчет урона

Commit:
добавлены атрибуты персонажа и их описание
дополнено описание арактеристик

Commit:
добавлен файл с расами

Commit:
удалено описание рас

Nikas10 commented on rules.txt:
```diff
@@ -25,32 +15,51 @@
+может блокировать атаки определенного списка других типов оружия. Щит позволяет блокировать урон от любого вида оружия. Измеряется в процентах. Зависит от свойств оружия.
+    h.Шанс уклонения - определяет вероятность, с которой персонаж уклонится от удара. Для уклонения от некоторых ударов могут потребоваться особые условия. 
+Щит не позволяет уклоняться от атак. Измеряется в процентах. Зависит от ловкости. Выше ловкость - больше вероятность уворота.
+    i.Прочность тела - определяет, насколько трудно нарушить целостность тела персонажа и повышает порог для нанесения критического удара. 
```
зачем нужен этот стат? почему нельзя обойтись хп / защитой?

Nikas10 commented on rules.txt:
```diff
@@ -25,32 +15,51 @@
+Щит не позволяет уклоняться от атак. Измеряется в процентах. Зависит от ловкости. Выше ловкость - больше вероятность уворота.
+    i.Прочность тела - определяет, насколько трудно нарушить целостность тела персонажа и повышает порог для нанесения критического удара. 
+Выше значение - выше прочность и значение порога. Зависит от выносливости. Выше выносливость - выше значение прочности.
+    j.Масса персонажа - определяет меру инертности персонажа. Зависит от веса экипировки и массы тела персонажа.
```
тоже пока сложно представить, зачем вводить стат. дрифт тяжелых персонажей на поворотах можно и на глаз сделать

Nikas10 commented on rules.txt:
```diff
@@ -25,32 +15,51 @@
+    e.Харизма - определяет способность персонажа общаться с другими персонажами.
+
+10)Основные характеристики персонажа:
+    a.Урон персонажа - определяет число очков здоровья, отнимаемых у цели при нанесении удара. Зависит от силы персонажа и эффектов, наложенных на персонажа. 
```
у нас есть урон персонажа, скалирующийся от силы. а что насчет заклинаний? у них константный урон или они скейлятся? предлагаю ввести "силу заклинаний"

Nikas10 commented on rules.txt:
```diff
@@ -25,32 +15,51 @@
+9)Основные атрибуты персонажа:
+    a.Сила - определяет физическую силу персонажа и эффективность физических воздействий на предметы. 
+    b.Ловкость - определяет гармоничность, плавность, резвость и грациозность персонажа.
+    c.Выносливость - определяет способность персонажа выдерживать различные нагрузки.
```
как думаешь, стоит ли добавлять атрибут силы воли, для сопротов?
алсо, в характеристиках отсутствуют иммунитеты, сопроты, предрасположенности и т.д.

Commit:
Атрибуты переименованы в базовые атрибуты.
Описание совершения действий изменено на более внятное.
Удалена прочность тела.
Урон зависит от атрибутов, а не от силы.

Gotus committed:
Инфраструктурные  изменения.
Описание базовых атрибутов вынесено в отдельный файл.
Описание характеристик вынесено в отдельный файл.

Gotus committed:
Первая версия правил боя.

Gotus committed:
Описание типов урона

Gotus committed:
Дополнены боевые механики

VladLeonov commented on damage.txt:
```diff
@@ -0,0 +1,6 @@
+    а) Физический урон. Наносится с помощью физических воздействий.
+    б) Магический урон. Наносится с помощью магических способностей.
+    
+    Если урон наносится с помощью магии, то его величина зависит от интеллекта, в противном случае - от силы атакующего.
```
Перебрать источники урона и посмотреть, все ли подходят под эти 2 категории или нужно еще (маловероятно).
Решить, нужны ли типы урона как стихии (вспомнить Card Hunter-овские режущий, огненный и т.д).

Составить примерные (простые) формулы урона для всех его видов. Параллельно с этим выяснить все параметры для его расчета (параметры персонажа, оружия и пр.)
Определить, какие параметры на расчеты каких типов урона влияют.

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+Основные характеристики персонажа:
+    a.Урон персонажа - определяет число очков здоровья, отнимаемых у цели при нанесении удара. Урон зависит от базовых атрибутов
+ персонажа. Чем выше значение, тем выше бонус к урону. Урон персонажа может изменяться благодаря эффектам, наложенным на него. 
```
А персонажа ли это характеристика? Или заклинания / оружия (голые руки - тоже оружие)?

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+    d.Максимальный переносимый вес - определяет максимальную массу, которую может перенести персонаж. Чем выше значение, тем выше переносимая масса. Зависит от силы 
+и выносливости персонажа.
+    e.Скорость передвижения - определяет время, затраачиваемое персонажем на перемещение до указанной точки. Скорость передвижения отряда равна скорости персонажа, 
+у которого она минимальна. Зависит от ловкости персонажа и наложенных эффектов. Чем выше ловкость, тем выше значение.
```
Нужны ли нам они?

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+    a.Урон персонажа - определяет число очков здоровья, отнимаемых у цели при нанесении удара. Урон зависит от базовых атрибутов
+ персонажа. Чем выше значение, тем выше бонус к урону. Урон персонажа может изменяться благодаря эффектам, наложенным на него. 
+    b.Заряды магии - используются для сотворения заклинаний. Имеют несколько уровней. Некоторые заклинания требуют для произнесения заряды различных уровней.
+Могут быть восстановлены за счёт отдыха в городе или эффектов. Число зарядов зависит от интеллекта персонажа и эффектов, наложенных на персонажа. 
```
Убрать ЭФФЕКТЫ отовсюду. Вынести в отдельный файл, если это необходимо. Со словами "могут влиять на все".

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+    e.Скорость передвижения - определяет время, затраачиваемое персонажем на перемещение до указанной точки. Скорость передвижения отряда равна скорости персонажа, 
+у которого она минимальна. Зависит от ловкости персонажа и наложенных эффектов. Чем выше ловкость, тем выше значение.
+    f.Скорость атаки - определяет количество ударов, совершённых за одну атаку. Чем выше значение, тем выше частота атак. Зависит от ловкости и наложенных на персонажа
+эффектов.
```
Пересмотреть механику, она сложная и важная. Возможно заменить на инициативу или броски на доп ход для быстрых персонажей.

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+    f.Скорость атаки - определяет количество ударов, совершённых за одну атаку. Чем выше значение, тем выше частота атак. Зависит от ловкости и наложенных на персонажа
+эффектов.
+    g.Шанс блока - определяет вероятность, с которой персонаж заблокирует урон. Для блокирования некоторых ударов могут потребоваться особые условия. Каждый вид оружия
+может блокировать атаки определенного списка других типов оружия. Щит позволяет блокировать урон от любого вида оружия. Измеряется в процентах. Зависит от свойств оружия.
```
"Что блочится чем" - можно свести на лор/здравый смысл, а не добавлять категории типов оружия к блокам.
Подумать, все ли блочится щитом (гранаты там), и стоит ли использовать проценты.

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+    g.Шанс блока - определяет вероятность, с которой персонаж заблокирует урон. Для блокирования некоторых ударов могут потребоваться особые условия. Каждый вид оружия
+может блокировать атаки определенного списка других типов оружия. Щит позволяет блокировать урон от любого вида оружия. Измеряется в процентах. Зависит от свойств оружия.
+    h.Шанс уклонения - определяет вероятность, с которой персонаж уклонится от удара. Для уклонения от некоторых ударов могут потребоваться особые условия. 
+Щит не позволяет уклоняться от атак. Измеряется в процентах. Зависит от ловкости. Выше ловкость - больше вероятность уворота.
```
Почему щит не позволяет?

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+может блокировать атаки определенного списка других типов оружия. Щит позволяет блокировать урон от любого вида оружия. Измеряется в процентах. Зависит от свойств оружия.
+    h.Шанс уклонения - определяет вероятность, с которой персонаж уклонится от удара. Для уклонения от некоторых ударов могут потребоваться особые условия. 
+Щит не позволяет уклоняться от атак. Измеряется в процентах. Зависит от ловкости. Выше ловкость - больше вероятность уворота.
+    i.Масса персонажа - определяет меру инертности персонажа. Зависит от веса экипировки и массы тела персонажа.
```
Ни нада

VladLeonov commented on characteristics.txt:
```diff
@@ -0,0 +1,24 @@
+выше значение инициативы.
+    k.Здоровье персонажа - определяет число очков здоровья персонажа. Более высокое значение соответствует более высокому количеству очков здоровья. Зависит от выносливости
+персонажа и наложенных эффектов. Значение характеристики увеличивается с ростом выносливости.
+    l.Рост персонажа - определяет размеры персонажа. Зависит от расы персонажа.
```
Ни нада 2
Только для лора

VladLeonov commented on basicAttributes.txt:
```diff
@@ -0,0 +1,11 @@
+    d.Интеллект - определяет умственную и магическую силу персонажа. Если атрибут станет ниже нуля, персонаж утратит сознание, 
+но сохранит тело.
+    e.Харизма - определяет способность персонажа общаться с другими персонажами. Если атрибут станет ниже нуля, 
+пол персонажа изменится.
```
Нужно ли вдобавок Восприятие?
Нужны ли дебаффы на негатив? Такие ли?

VladLeonov commented on battleMechanics.txt:
```diff
@@ -0,0 +1,31 @@
+получилось одинаковое значение, то очерёдность их хода зависит от различных условий. Если бойцы сражаются на одной стороне, они могут
+согласовать порядок хода, либо ходить по правилам для противников. Если бойцы являются противниками, первым ходит боец с наивысшей
+ловкостью. Если ловкость одинакова, делается дополнительный бросок для каждого из персонажей. Тот, кто выбросил наибольшее значение,
+ходит первым.
```
Заменит стоит на более простую схему. Например:
"В начале хода баффнутыми ловкостью кубиками определяется последовательность хода в бою среди всех участников боя. После своего хода можно бросить кубик на шанс дополнительного хода (можно несколько раз). Шанс зависит от ловкости и падает с каждым броском. Шанс восстанавливается на следующий свой "недополнительный" ход."

VladLeonov commented on battleMechanics.txt:
```diff
@@ -0,0 +1,31 @@
+
+Перемещение во время боя
+    Дистанция, на которую может переместиться персонаж за один ход, зависит от массы персонажа, его силы, выносливости 
+и наложенных эффектов. Масса персонажа уменьшает дальность, базовые атрибуты - увеличивают.
```
Дописать.

Gotus committed:
Убраны лишние эффекты

Gotus committed:
Убраны лишние эффекты из правил боя

Gotus committed:
Добавлено восприятие

Gotus committed:
Описание урона перенесено в описание боя

Gotus committed:
Исправлена нумерация

Gotus committed:
Добавлены формулы для рассчета урона

Gotus committed:
Добавлены механики защиты

Gotus committed:
Изменены правила определения порядка хода

Gotus committed:
Изменена механика защиты

Gotus committed:
Изменено оформление

VladLeonov commented on damage.txt:
```diff
@@ -13,9 +13,8 @@
+которое блокируется. Тело персонажа разделено на 4 области, каждая из которых обладает своей защитой:
         a.Голова. Защита этой области повышается за счёт шлема.
         b.Тело. Защита этой области повышается за счёт доспехов, надеваюзихся на торс.
         c.Руки. Повышается за счёт наплечников, налокотников и перчаток.
```
Не указано, на что влияет каждая из областей. Мы суммируем их защиту?

Gotus committed:
Уточнена механика защиты

Gotus committed:
Шанс блока перенесён в боевые механики

Nikas10 commented on basicAttributes.txt:
```diff
@@ -0,0 +1,14 @@
+персонаж не сможет воздействовать на предметы. Когда на персонажа с отрицательной силой действует другой персонаж, то масса и прочность
+первого считаются равными нулю.
+    b.Ловкость - определяет гармоничность, плавность, резвость и грациозность персонажа. Если атрибут станет ниже нуля, персонаж 
+станет неуязвимым неподвижным объектом. Персонаж может менять местоположение, будучи пеердвинутым другим персонажем.
```
неуязвимый это bullshit и дыра для абузов. просто неподвижный

Nikas10 commented on basicAttributes.txt:
```diff
@@ -0,0 +1,14 @@
+    d.Интеллект - определяет умственную и магическую силу персонажа. Если атрибут станет ниже нуля, персонаж утратит сознание, 
+но сохранит тело. Любой магический урон, получаемый персонажем, будет критическим.
+    e.Харизма - определяет способность персонажа общаться с другими персонажами. Если атрибут станет ниже нуля, 
+пол персонажа изменится. Персонаж с отрицательной харизмой будет делать всё, что ему скажут другие персонажи.
```
пол? нани? это не звучит как дебафф для отрицательного стата

Nikas10 commented on characteristics.txt:
```diff
@@ -0,0 +1,15 @@
+    a.Заряды магии - используются для сотворения заклинаний. Имеют несколько уровней. Некоторые заклинания требуют для произнесения заряды различных уровней.
+Могут быть восстановлены за счёт отдыха в городе. Число зарядов зависит от интеллекта персонажа. 
+Чем выше интеллект, тем больше зарядов магии каждого уровня.
+    b.Максимальный переносимый вес - определяет максимальную массу, которую может перенести персонаж. Чем выше значение, тем выше переносимая масса. Зависит от силы 
```
нужен ли нам макс вес? может сделать слоты или вообще не париться?

Nikas10 commented on damage.txt:
```diff
@@ -0,0 +1,22 @@
+
+Физический урон в ближнем бою = базовый урон оружия + 2 * сила
+Физический урон в дальнем бою = базовый урон оружия + восприятие
+Магический урон = базовый урон оружия + интеллект
```
если стата является преобладающим фактором расчета дамага, у миликов огромное преимущество.
мб сделать модификатор поменьше?

Nikas10 commented on damage.txt:
```diff
@@ -0,0 +1,22 @@
+        b.Тело. Защита этой области повышается за счёт доспехов, надеваюзихся на торс.
+        c.Руки. Повышается за счёт наплечников, налокотников и перчаток.
+        d.Ноги. Повышается за счёт ботинок и набедренников.
+    При нанесении урона по определённой области для рассчета урона берётся защита этой области.
```
можно ещё отдельно расписать, какие рюшечки может носить персонаж и где (кольца там всякие, триньки...)

Gotus committed:
Изменены механики ловкости и харизмы

Gotus committed:
Переносимый вес заменн на число слотов

Gotus committed:
Добавлен план

Gotus committed:
Удалена масса персонажа

Gotus committed:
Уточнена механика эффектов

Gotus committed:
Исправлена опечатка
